src/types/bookmarks.ts

```diff
@@ -0,0 +1,23 @@
+export enum BookmarkTypes {
+  folder = 'folder',
+  shabad = 'file',
+  bani = 'book-open'
```

Comment:
I'm guessing these refer to the corresponding icons. I wouldn't define these here, but instead create a `const bookmarkIcons = { [ BookmarkTypes.folder ]: folder, ...etc }` in the place that you wish to do the rendering